src/java-runtime/utils/upstreamApi.ts

```diff
@@ -42,51 +42,52 @@ export async function resolveRequirements(): Promise<{
             javaVersion = await getMajorVersion(javaHome);
             toolingJre = javaHome;
```

Comment:
On vscode-java, it's a different about picking java.home as tooling jre.

![image](https://user-images.githubusercontent.com/14052197/150940352-d9c96f66-798f-4f5b-9736-ef4c57dc6e9e.png)
See [source code on vscode-java](https://github.com/redhat-developer/vscode-java/blob/de65a586d36a79034a533d564f470b3a2754c1fe/src/requirements.ts#L56-L59)

Reply:
so in vscode-java, `java.home` will not be picked up as jdtls runtime, right?

Reply:
BTW, here the logic is to determine whether to pop up "configure Java runtime" for the tooling runtime. Different from upstream vscode-java, here in pack, we simply detect whether toolingJre is valid. Default project runtime `javaHome` is not important.

Reply:
> so in vscode-java, java.home will not be picked up as jdtls runtime, right?

Yes, here is the order to look for a tooling JRE:
"java.jdt.ls.java.home" > embedded jre > "java.home"

Reply:
Thanks, updated.